modules/programs/oh-my-posh.nix

```diff
@@ -41,6 +43,18 @@ in
       '';
     };
 
+    configFile = lib.mkOption {
+      type = lib.types.nullOr lib.types.str;
```

Comment:
Should support a path itself for an option that says its for configuring a file.

Reply:
I'll work on making this support an explicit path.